### PR TITLE
Add secretk to Developer Tools

### PR DESCRIPTION
**Pull request by luca992**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,6 +212,7 @@ Tools for interacting with the secret network and for writing Secret Contracts
 * [Secret Py](https://pypi.org/project/secret-sdk/) - Python library for interacting with the network
 * [secret.js](https://github.com/scrtlabs/secret.js) - JavaScript library for interacting with the network
     ([examples](https://github.com/scrtlabs/SecretJS-Templates))
+* [secretk](https://github.com/eqoty-labs/secretk) - Kotlin Multipltform (JVM, Android, iOS, macOS, JS) library for interacting with the network
```
Comment (reuvenpo) [inline README.md]:
```suggestion
* [secretk](https://github.com/eqoty-labs/secretk) - Kotlin Multi-platform (JVM, Android, iOS, macOS, JS) library for interacting with the network
```

**Review by reuvenpo — CHANGES_REQUESTED** [review]
Thanks for the PR!
Please fix the small typo though :)

File: README.md (hunk 1 above)
Comment (luca992) [reply README.md]: That's not normally how multiplatform is spelt:
https://kotlinlang.org/docs/multiplatform.html

But, up to you. Just lmk to confirm.

Comment (reuvenpo) [reply README.md]: The dash isn't important to me, but you were missing an `a` between `pl` and `tform`

Comment (reuvenpo) [reply README.md]: On another look, I'd also like it to be one line higher, before secretjs, after secretpy

Comment (luca992) [reply README.md]: ooh. Wow I completely overlooked that. got it.

**Review by reuvenpo — APPROVED** [review]

**Pull request merged**